src/components/PostComment.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import { postComment } from "../utils/api";
+import moment from "moment";
+
+export const PostComment = ({ review_id }) => {
+  const [body, setBody] = useState("");
+  let username = "jessjelly";
+
+  const handleSubmit = (e) => {
+    if (body.length < 10 || username === "") {
+      e.preventDefault();
+      alert("Comments must be 10 characters or more");
```

Comment:
`alert` is not the friendliest user feedback you can give, it's disruptive, it's a blocking function and it can be disabled by the client so users could miss these messages!

Everywhere you've used alert i'd recommend rendering something on the page instead.